game-rules.tex

```diff
@@ -4,16 +4,25 @@ \section{Game Rules}
 \begin{enumerate}
   \item The game, called \emph{\gamename}, is played in the arena defined in
         Specification~\ref{spec:arena}. The objective is to collect tokens and
-        deposit them in the teams scoring zone.
-  \item The arena contains 16 tokens.
-  \item There are four scoring zones in the arena, arranged in a $2\times2$
-        grid, with each robot being allocated a single scoring zone. The centre
-        of the arena (approximately a quarter of each scoring zone) is raised.
+        deposit them in scoring zones.
+  \item The arena contains 24 tokens.
+  \item There are nine scoring zones in the arena, arranged in a $3\times3$
+        grid. The central scoring zone is raised.
+  \item A scoring zone is considered to be `controlled' by a team if and only if:
+    \begin{enumerate}
+      \item the team has tokens in the zone.
+      \item no other team has a greater than, or equal to number of tokens in the zone.
+    \end{enumerate}
+  \item A `region' is defined as follows:
+    \begin{enumerate}
+      \item A `region' is a group of directly adjacent scoring zones controlled by the same team.
+      \item The size of a `region' is calculated by counting the number of scoring zones within the region.
+      \item If the central scoring zone is within a `region', the size of the region is increased by one.
```

Comment:
by an *additional* one, or otherwise clarify this is a bonus on top of the normal region scoring

Reply:
This section is going to be completely reworded as discussed last night.